include/seastar/fs/block_device.hh

```diff
@@ -91,7 +91,6 @@ public:
     future<> close() override {
         return _file.close();
     }
-
```

Comment:
code style request - either drop this change or move it to the previous commit. This commit should only contain changes to the unit test.